erpnext/projects/doctype/project_team_members/project_team_members.json

```diff
@@ -0,0 +1,37 @@
+{
+ "creation": "2020-11-11 03:08:01.188509",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "employee",
```

Comment:
- Lets change this to User.
- A Website User can also be assigned a project, he need not be an employee necessarily.